sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -814,4 +816,50 @@ object DDLUtils {
       }
     }
   }
+
+  /**
+   * ALTER TABLE ADD COLUMNS command does not support temporary view/table,
+   * view, or datasource table with text, orc formats or external provider.
+   */
+  def verifyAlterTableAddColumn(
```

Comment:
Since this checking is only used in `AlterTableAddColumnsCommand `, we do not need to move it here